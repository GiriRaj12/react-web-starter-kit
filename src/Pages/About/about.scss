.basic-view{
    text-align: center;
    padding: 40px;
    font-size: large;
    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
    width: 50%;
    margin: 50px auto;
    box-shadow: 0 0 5px 0px rgb(238, 238, 238)
}