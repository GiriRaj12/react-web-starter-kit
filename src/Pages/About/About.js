import React from 'react';
import './about.scss';

function About() {
    return (<div className='basic-view'>
        Created to kick start project easier.
    </div>)
}

export default About;