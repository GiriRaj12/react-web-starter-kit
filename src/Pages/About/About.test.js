import React from 'react';
import renderer from 'react-test-renderer';
import About from './About.js';

test('About Snapshot test ', () => {
    const render = renderer.create(<About></About>)
    expect(render.toJSON()).toMatchSnapshot();
})