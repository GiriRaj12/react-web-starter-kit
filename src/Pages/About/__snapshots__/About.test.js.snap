// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Snapshot test  1`] = `
<div
  className="basic-view"
>
  Created to kick start project easier.
</div>
`;
